src/ape/contracts/base.py

```diff
@@ -842,6 +843,17 @@ def at(self, address: AddressType, txn_hash: Optional[str] = None) -> ContractIn
             address, self.contract_type, txn_hash=txn_hash
         )
 
+    @cached_property
+    def source_path(self) -> Optional[Path]:
+        """
+        Returns the path to the local contract if determined that this container
+        belongs to the active project.
+        """
+        contract_name = self.contract_type.name
+        contract_type = self.project_manager._get_contract(contract_name)
+        if contract_type and self.contract_type:  # == contract_type:
```

Comment:
these don't come out to the same thing so I commented out `== contract_type`

Reply:
can you do maybe something like self.contract_type.runtime_bytecode.bytecode == contract_type.runtime_bytecode.bytecode